recipe/meta.yaml

```diff
@@ -22,23 +22,24 @@ requirements:
     - pip
     - setuptools
     - wheel
-    - numpy >=1.23
+    - numpy {{ numpy }}
   run:
     - python
     - numpy >=1.23
-    - pandas
     - numexpr
-    - matplotlib-base
     - wheel
-    - canmatrix[arxml,dbc] >=0.9.5
-    - cchardet
-    - chardet
-    - lxml
+    - pandas
+    - canmatrix >=0.8
     - natsort
+    - lxml
     - lz4
-    - typing_extensions
-    - python-isal [x86_64]
     - python-dateutil
+    #optional dependencies
+    - matplotlib-base #fallback for Signal plotting
+    - cchardet #to detect non-standard Unicode encodings
+    - chardet  #to detect non-standard Unicode encodings
+    - typing_extensions #for python <3.10
```

Comment:
`typing_extensions  # [py<310]`?

Reply:
hmm, this fails the build, I am not sure why CF guys add this when it is not in the upstream but it is not just py <310